cypress/e2e/requests/request-moved-to-fulfillment-in-progress.cy.js

```diff
@@ -0,0 +1,178 @@
+import TopMenu from '../../support/fragments/topMenu';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import SwitchServicePoint from '../../support/fragments/settings/tenant/servicePoints/switchServicePoint';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import getRandomPostfix from '../../support/utils/stringTools';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import TitleLevelRequests from '../../support/fragments/settings/circulation/titleLevelRequests';
+import Requests from '../../support/fragments/requests/requests';
+import RequestDetail from '../../support/fragments/requests/requestDetail';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import InTransit from '../../support/fragments/checkin/modals/inTransit';
+
+describe('Title Level Request. Request queue. TLR', () => {
+  let userData = {};
+  let requestId;
+  const testData = {
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    servicePoint2: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    barcode: generateItemBarcode(),
+    title: `Instance_C350425_${getRandomPostfix()}`,
+  };
+  const patronGroup = {
+    name: 'groupTLR' + getRandomPostfix(),
+  };
+
+  before('Preconditions:', () => {
+    cy.getAdminToken()
+      .then(() => {
+        ServicePoints.createViaApi(testData.servicePoint);
+        ServicePoints.createViaApi(testData.servicePoint2);
+        cy.loginAsAdmin({
+          path: SettingsMenu.circulationTitleLevelRequestsPath,
+          waiter: TitleLevelRequests.waitLoading,
+        });
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+        testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+          testData.materialType = materialTypes.name;
+        });
+      })
+      .then(() => {
+        InventoryInstances.createFolioInstanceViaApi({
```

Comment:
better to use the new approach for creating instances
example `cypress/e2e/loans/multiple-loans.cy.js`

Reply:
done)